doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -689,6 +689,31 @@ Note also that in the absence of a ``as ident``, ``as strict pattern`` or
 ``as pattern`` :n:`@syntax_modifier`\s, the default is to consider sub-expressions occurring
 in binding position and parsed as terms to be ``as ident``.
 
+Binders bound in the notation and parsed as general binders
```

Comment:
Even after 10 minutes I still don't understand this--what's been added, when it is useful.  The existing description of Notation and syntax_modifier need considerable work (separate from this PR).  Hope you can make this better.

Reply:
There is two levels of binding. The bindings in terms, like in `fun`, `forall`, `let` and the bindings of subterm expressions by variables of a notation. The second are at a meta-level since they operate on terms which themselves have binders at the (lower) term-level (also called object-level in logic).

In particular, one can bind at the meta-level variables which play a role of binder at the term-level. Otherwise said, "Binders bound" should be read as "Term-binders bound at the notation level".